src/shapes/sprite.js

```diff
@@ -14,16 +14,18 @@ function SpriteShape(params) {
     dx: params.x,
     dy: params.y,
     speed: properties.speed,
-    animation: null
+    update: null,
+    animation: properties.animation === true ? true : false,
```

Comment:
So, if the client set:

```js
{
  animation: "teste"
}
```

Animation work, because **"teste"** is equal **true**.